source/community.applications/usr/local/emhttp/plugins/community.applications/javascript/helpers.js

```
@@ -2,395 +2,994 @@
+			if ( ! $(".sweet-overlay").is(":visible") ) {
+				mySpinner();
+			}
+
+		}
+		postCount++;
+	}
+
+	$.post(execURL,options).done(function(result) {
+		if (result.script) {
+			try {
+				eval(result.script);
```

Biome (2.4.13)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)